tests/PKSim.Tests/IntegrationTests/ContextForIntegration.cs

```diff
@@ -49,12 +49,9 @@ public override void GlobalContext()
             container.Register<IDisplayUnitRetriever, CLIDisplayUnitRetriever>();
             container.Register<IOntogenyFactorsRetriever, OntogenyFactorsRetriever>();
             container.Register<ISimulationConstructor, SimulationConstructor>();
-
             container.RegisterImplementationOf(A.Fake<IProgressUpdater>());
             container.RegisterImplementationOf(A.Fake<IDialogCreator>());
             container.RegisterImplementationOf(A.Fake<IHistoryManager>());
-            container.RegisterImplementationOf(A.Fake<IWorkspace>());
```

Comment:
This was wrong here. Not sure why it worked before. We want to use a REAL workspace object and not a fake!